SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -1,32 +1,47 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
-# How to port a native Unreal project to the Unreal GDK
-
-> This workflow is very subject to change. We're aware that it's not optimized, and one of our priorities is to improve iteration times.
+# How to port a native Unreal project to the SpatialOS GDK for Unreal
 
 ## Prerequisites
 
 <%(TOC)%>
 
-Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
-> To speed up build times, install IncrediBuild, FastBuild, or another build distributor.
-
-## Setting up the project structure
-1. In a terminal window, navigate to the root directory of the SpatialOS Unreal GDK repository you cloned and run `Setup.bat`. This requires authorization with your SpatialOS account.
-1. Open the `StarterProject` repo and copy everything except the `\Game\` folder alongside your `<GameRoot>` folder. 
-Your folder structure should now resemble -
-```
-\<ProjectRoot>\ci\
-\<ProjectRoot>\<GameRoot>\
-\<ProjectRoot>\spatial\
-\<ProjectRoot>\ProjectPaths.bat
-\<ProjectRoot>\LaunchClient.bat 
-etc...
-```
-1. Open **`ProjectPaths.bat`** for editing and replace `Game` with `<GameRoot>`, and `StarterProject` with the name of your uproject (which we'll refer to as `<YourProject>`).
-
-## Adding the SpatialGDK module to your project
-1. In your project's `*.build.cs` file, add "SpatialGDK" to the `PublicDependencyModuleNames`.
+Before porting your project, you need to install and set up the SpatialOS GDK: follow the [set up guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
```

Comment:
Adding note to self to update this. Please could we have a "TODO: update the setup link" in the comment here.